lib/web_apis/web_interface.es6.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  name: 'WebInterface',
+  package: 'org.chromium.apis.web',
+  ids: ['interfaceKey'],
+  properties: [
+    {
+      class: 'String',
+      name: 'interfaceName',
+      documentation: `The name of a web interface, extracted from
+        apiExtractor.`,
+      required: true,
+      final: true,
+    },
+    {
+      class: 'String',
+      name: 'apiName',
+      documentation: `The name of an API. APIs are identified as meaningful
+        properties of an interface.`,
```

Comment:
Can we add a TODO to link out to some public documentation on precisely how we define and gather "meaningful properties"?